packages/ppx/src/css_to_emotion.re

```diff
@@ -90,23 +95,24 @@ and render_media_query = (ar: At_rule.t): Parsetree.expression => {
           loc_end: last_value_loc.Location.loc_end,
         };
         let value = source_code_of_loc(loc);
-        let () =
-          switch (Declarations_to_emotion.parse_declarations(ident, value)) {
+        let exprs =
+          switch (Declarations_to_string.parse_declarations(ident, value)) {
           | Error(`Not_found) =>
             grammar_error(ident_loc, "unsupported property: " ++ ident)
           | Error(`Invalid_value(_error)) =>
             grammar_error(loc, "invalid value")
-          | Ok(_) => ()
+          | Ok(exprs) => exprs
           };
-        source_code_of_loc(complete_loc);
+          List.fold_left(concat(~loc), acc, exprs);
```

Comment:
Sweet! 

Do we handle the parsing errors before?

Reply:
`Declarations_to_string` will raise if the value is invalid or the property is not found, and i just commited a fix to raise when the prelude is empty, preventing rules like this `@media {color: blue}` to be valid